scripts/build_automation/bitrise_workflows/tests.yml

```diff
@@ -241,6 +241,15 @@ workflows:
             brew tap thii/xcbeautify https://github.com/thii/xcbeautify.git
             brew install awscli swiftlint jq xcbeautify mxcl/made/swift-sh
             brew install Caskroom/cask/soundflower
+    - script:
+        title: Grant simulator microphone permissions
+        run_if: .IsCI
+        inputs:
+        - content: |-
+            sudo sqlite3 ~/Library/Application\ Support/com.apple.TCC/TCC.db "
+                INSERT INTO access VALUES('kTCCServiceMicrophone','com.apple.CoreSimulator.SimulatorTrampoline',0,1,1,
+                X'fade0c00000000480000000100000006000000020000002b636f6d2e6170706c652e436f726553696d756c61746f722e53696d756c61746f725472616d706f6c696e650000000003',
+                NULL,NULL,'UNUSED',NULL,NULL,1561614941);"
```

Comment:
See https://discuss.bitrise.io/t/mojave-sip-prompt-for-cmd-bridge-and-terminal/10102/2